### PR TITLE
Ohada balance sheet enhancement

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR enhance the ohada balance sheet for getting data from the period zero of the next fiscal year.

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -5,7 +5,7 @@ PORT=8080
 DB_HOST='localhost'
 DB_USER='bhima'
 DB_PASS='HISCongo2013'
-DB_NAME='bhima_test'
+DB_NAME='imck'
```
Comment (lomamech) [inline .env.development]: @mbayopanda restore 'bhima_test' like Db name and not IMCK

**Comment by lomamech** [comment]
image

The display of the 2017 Balance Sheet data is not correct

File: .env.development (hunk 1 above)
Comment (jniles) [reply .env.development]: Also, did you know that you can change the environmental variables globally by just prefixing the script?

```sh
# This should make the DB_NAME the variable
DB_NAME="imck" npm run dev
```

**Comment by lomamech** [comment]
@mbayopanda According to @patsonjunior 'Patrick Kianzuaku' the Passive Part should be displayed in absolute value as for the report operating account with the exception of CI 
"Resultat Net de l'exercice" which can be either debtor or creditor

**Comment by mbayopanda** [comment]
@lomamech could you review this PR please

**Pull request merged**